static/app/views/automations/hooks/index.tsx

```diff
@@ -67,6 +71,45 @@ export function useAutomationQuery(automationId: string) {
   });
 }
 
+export const makeAutomationFireHistoryQueryKey = ({
+  orgSlug,
+  id,
+  cursor,
+  limit,
+  query,
+}: {
+  id: string;
+  orgSlug: string;
+  cursor?: string;
+  limit?: number;
+  query?: string;
+}): ApiQueryKey => [
+  `/organizations/${orgSlug}/workflows/${id}/group-history/`,
+  {query: {query, per_page: limit, cursor}},
+];
+
+interface UseAutomationFireHistoryQueryOptions {
+  id: string;
```

Comment:
we're passing this `id` around a bit might help to give it a full name
```suggestion
  workflowId: string;
```

Reply:
or automationId or whatever